CHANGELOG.rst

```diff
@@ -18,6 +18,8 @@ Unreleased
 
 **Fixed**
 
+* Docs for analytical kinematics
```

Comment:
The doc update is not a code change, so, it doesn't need to be in the changelog. 
However the handling of `acm` does, so I would change this line to explain the acm bit.